tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -165,22 +165,24 @@ spec:
         echo -n "${mustPublishIndexImage}" | tee "$(results.mustPublishIndexImage.path)"
         echo -n "${fbc_opt_in}" | tee "$(results.isFbcOptIn.path)"
 
-        # Whether the index image will be published
+        jq '.reason // "Unset"'  <<< "${conditions}" | tee "$(results.requestReason.path)"
+        jq '.message // "Unset"' <<< "${conditions}" | tee "$(results.requestMessage.path)"
+
+        jq -r '.indexImageDigests' <<< "${results}" |  tee "$(results.indexImageDigests.path)"
+
+        jq -r '.iibLog' <<< "${results}"
+        RC="$(jq -r '.exitCode' <<< "${results}")"
+
+        # Summarize what happened for the human user.
```

Comment:
Maybe we should also mention, in cases when `fbc_opt_in=true` and `stagedIndex=true`, that this will also not be published. I remember seeing some asks about this. 

Something like "Your image has fbc_opt_in set but this is a staging index and it won't be published".

Reply:
... and then the correct call to action to resolve this (if publishing WAS intended or the target wasn't meant to be stage). 

The Konflux EC violations are a great example of how to provide detailed and helpful suggestions to fix problems. Why not follow that model?

Reply:
> Maybe we should also mention, in cases when fbc_opt_in=true and stagedIndex=true, that this will also not be published. I remember seeing some asks about this.

I think the current logic will already achieve this.

If fbc_opt_in is true, then the first conditional won't hit. The second conditional won't hit. And, that leaves the third conditional which will print:

> Index image will not be published because this is a staging release.

(Nick's comment aside), I think this is good enough.